pkg/helpers/helpers.go

```diff
@@ -600,7 +600,7 @@ func LoadClientConfigFromSecret(secret *corev1.Secret) (*rest.Config, error) {
 func DetermineReplica(ctx context.Context, kubeClient kubernetes.Interface, mode operatorapiv1.InstallMode) int32 {
 	// For hosted mode, there may be many cluster-manager/klusterlet running on the management cluster,
 	// set the replica to 1 to reduce the footprint of the management cluster.
-	if mode == operatorapiv1.InstallModeDetached {
+	if mode == operatorapiv1.InstallModeHosted {
```

Comment:
it is fine if it is read from config, but detached needs to be considered if it is read from klusterlet resource

Reply:
Sure, updated.